Resources/Prototypes/Entities/Objects/Devices/pda.yml

```
@@ -106,6 +106,7 @@
         type: InstrumentBoundUserInterface
       enum.HealthAnalyzerUiKey.Key:
         type: HealthAnalyzerBoundUserInterface
+  - type: RemoteStore
```

```suggestion
  - type: RemoteStore #PDAs can access uplinks so they need this component.
```
comments are like the flow of water. As one pool drains, another fills.